examples/qml_features/rust/src/properties.rs

```diff
@@ -21,20 +21,20 @@ pub mod qobject {
         // ANCHOR: book_properties_signature
         #[qobject]
         #[qml_element]
-        #[qproperty(bool, connected)]
-        #[qproperty(QUrl, connected_url)]
-        #[qproperty(QUrl, previous_connected_url)]
-        #[qproperty(QString, status_message)]
+        #[qproperty(bool, connected, READ, NOTIFY = on_changed)]
+        #[qproperty(QUrl, connected_url, READ, WRITE = set_url, NOTIFY = on_changed, RESET = reset_url)]
+        #[qproperty(QUrl, previous_connected_url, READ, NOTIFY = on_changed)]
+        #[qproperty(QString, status_message, READ, NOTIFY = on_changed)]
         type RustProperties = super::RustPropertiesRust;
         // ANCHOR_END: book_properties_signature
 
-        /// Connect to the given url
-        #[qinvokable]
-        fn connect(self: Pin<&mut RustProperties>, mut url: QUrl);
+        /// Custom on changed signal, used for all the properties
+        #[qsignal]
+        fn on_changed(self: Pin<&mut RustProperties>);
 
-        /// Disconnect from the stored url
-        #[qinvokable]
-        fn disconnect(self: Pin<&mut RustProperties>);
+        fn set_url(self: Pin<&mut RustProperties>, url: QUrl);
```

Comment:
add some doc lines to the set_url and reset_url so people following the example can understand them

Reply:
Ah i see the implementation has them and `missing_docs` doesn't appear to complain, so either copy them to the same as below or leave them out :-)

Reply:
they're on the implementation but will do